crs_motion_planning/src/motion_planning_server.cpp

```diff
@@ -549,6 +559,62 @@ class MotionPlanningServer : public rclcpp::Node
     return;
   }
 
+  void moveURBase(std::shared_ptr<crs_msgs::srv::RobotPositioner::Request> request,
```

Comment:
This is minor but I prefer non vendor/model specific names for methods, variables, etc

Reply:
That makes sense, I think this was also the service we had discussed last week that we might want to relocate to somewhere in the application node as opposed to the motion planning server